OrbitGrpcProtos/services.proto

```diff
@@ -50,6 +50,7 @@ message GetProcessMemoryResponse {
 
 message GetDebugInfoFileRequest {
   string module_path = 1;
+  reserved 2;
```

Comment:
lets have reserved list in the first line, this way we won't need to move it around

Reply:
just to clarify - this is a list so if we have 2 reserved values it becomes reserved 1, 2;

Reply:
okay, makes sense - done, will merge now